moveit_ros/planning/constraint_sampler_manager_loader/src/constraint_sampler_manager_loader.cpp

```diff
@@ -70,27 +74,27 @@ class ConstraintSamplerManagerLoader::Helper
           constraint_samplers::ConstraintSamplerAllocatorPtr csa =
               constraint_sampler_plugin_loader_->createUniqueInstance(*beg);
           csm->registerSamplerAllocator(csa);
-          ROS_INFO("Loaded constraint sampling plugin %s", std::string(*beg).c_str());
+          RCLCPP_INFO(node_->get_logger(), "Loaded constraint sampling plugin %s", std::string(*beg).c_str());
         }
         catch (pluginlib::PluginlibException& ex)
         {
-          ROS_ERROR_STREAM("Exception while planning adapter plugin '" << *beg << "': " << ex.what());
+          RCLCPP_ERROR(node_->get_logger(), "Exception while planning adapter plugin '%s': %s",
+                       std::string(*beg).c_str(), ex.what());
         }
       }
     }
   }
 
 private:
-  ros::NodeHandle nh_;
+  rclcpp::Node::SharedPtr node_;
```

Comment:
I actually wasn't aware of `*::SharedPtr`. I would vote for using this instead of `std::shared_ptr` everywhere.